### PR TITLE
Add obtain test dir

### PR DESCRIPTION
**Pull request by xiuwang**
@openshift/devexp-qe Please help review, thanks

Passed locally http://pastebin.test.redhat.com/875383

### CODE REVIEW

File: features/build/stibuild.feature [hunk 1]
```diff
@@ -4,7 +4,7 @@ Feature: stibuild.feature
     Given I have a project
     Given I obtain test data file "templates/<template>"
     And I run the :new_app client command with:
-      | file | <template> |
+      | file | tc498848-s2i.json |
```
Comment (akostadinov) [inline features/build/stibuild.feature:7]: this change doesn't look right

Other than that looks good.

Comment (xiuwang) [reply features/build/stibuild.feature:7]: The original snenario is a multiple one, now other cases have removed from 4.x, only this case is left. I don't want to change the automation field in polarion , just update this case instead

Comment (akostadinov) [reply features/build/stibuild.feature:7]: but few lines above you have `I obtain test data file "templates/<template>"` which depends on the parameter. It doesn't make sense to me. Updating polarion is just calling a simple script 
https://mojo.redhat.com/docs/DOC-1125811

Although I don't see why you'd need to change.

Comment (akostadinov) [reply features/build/stibuild.feature:7]: Ok, I've got it. Not ideal but anyway.

**Pull request merged**